Demo_2/index.html

```diff
@@ -0,0 +1,582 @@
+<!DOCTYPE html>
```

Comment:
In general, the same comments as for `Demo_1/index.html` apply to this file.

Reply:
As with _Demo\_1/index.html_ surely some place for improvements. But as it is legacy migration no real value to gain for the spent time